product_docs/docs/pem/9/pem_rel_notes/960_rel_notes.mdx

```diff
@@ -17,7 +17,7 @@ New features, enhancements, bug fixes, and other changes in PEM 9.6.0 include:
 | Enhancement  | Added an option to download the Capacity Manager report in JSON format.                                                                                                                                                                      |
 | Enhancement  | You can now specify a CA Certificate path when registering a Barman server using pemworker without having to specify a client certificate and key.                                                                                           |
 | Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead of 'none' when log statement parameter was left empty in the GUI resulting in failure of server restart.                                        |
-| Bug&nbsp;fix | Fixed an issue where audit manager was setting `edb_audit_statement` parameter to '' instead of 'none' when the log statement parameter was left empty in the GUI. This issue resulted in the failure of server restart.                     |
+| Bug&nbsp;fix | Fixed an issue where `os_info` probe was throwing error whenever timezone/timestamp were changed.                                                                                                                                            |
```

Comment:
Fixed an issue where `os_info` probe was throwing error for few timezone/timestamp.